src/DistributedFactorGraphs.jl

```diff
@@ -273,24 +274,28 @@ include("services/CompareUtils.jl")
 # Include the FilesDFG API.
 include("FileDFG/FileDFG.jl")
 
-include("CloudGraphsDFG/CloudGraphsDFG.jl")
-
 # Custom show and printing for variable factor etc.
 include("services/CustomPrinting.jl")
 
+# To be moved as necessary.
+include("Common.jl")
+
+# Big data extensions
+include("BigData/BigData.jl")
+
 function __init__()
     @require GraphPlot = "a2cc645c-3eea-5389-862e-a155d0052231" begin
         @info "Including Plots"
         include("DFGPlots/DFGPlots.jl")
         @reexport using .DFGPlots
     end
 
+    if get(ENV, "DFG_USE_CGDFG", "") == "true"
+        @info "Including CloudGraphsDFG"
+        @eval Base.include(DistributedFactorGraphs, joinpath(@__DIR__, "CloudGraphsDFG/CloudGraphsDFG.jl"))
+        @eval DistributedFactorGraphs @reexport using .CloudGraphsDFGs
+    end
```

Comment:
@dehann How do we want to do this?
The other way is to do the check at compile time, but then you have to manually trigger the compile if you want to change it.

Reply:
Not sure i understand the other way...?  But either way ENV variable is good/required, and its also fine to trigger new precompile if thats needed.

Reply:
Other way is not in `function __init__()`, so its pre-compiled.